src/octoprint/server/api/job.py

```diff
@@ -34,14 +34,16 @@ def controlJob():
 
 	tags = {"source:api", "api:job"}
 
+	user = current_user._get_current_object()
```

Comment:
Turn this into `current_user.get_name()` and do the PR against the `maintenance` branch (small feature, not a huge one that might break stuff, so fair game for 1.3.8) and this is a winner :)